staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Staking Precompile
+description: Moonbeam Parachain Staking Ethereum Solidity Precompile Interface Demo
+---
+
+# Staking Precompile
+
+![Staking Moonbeam Banner](/images/staking/staking-stake-banner.png)
+
+## Moonbeam Parachain-Staking
+
+A delegated proof of stake pallet recently debuted called [Parachain-Staking](https://github.com/PureStake/moonbeam/tree/master/pallets/parachain-staking/src), allowing token holders (nominators) to express exactly which collator candidates they would like to support and with what quantity of stake. The design of the Parachain-Staking pallet is such that it enforces shared risk/reward on chain between delegators and collators.
+
+The Staking module is coded in Rust and it is part of a pallet that is normally not accessible from the Ethereum side of Moonbeam. However, a Staking Precompile allows developers to access the staking features as a precompiled contract located at address 0x0000000000000000000000000000000000000800. The Staking Precompile is included in our [Moonbase Alpha v8 release](https://moonbeam.network/announcements/testnet-upgrade-moonbase-alpha-v8/).
```

Comment:
You made a variable for the address but also hardcoded it here.

Reply:
Speaking of addresses, this is the canonical place to find the list. Just an FYI, not that it needs to go in the docs.

https://github.com/PureStake/moonbeam/blob/master/runtime/moonbase/src/precompiles.rs#L78-L96